packages/ui-components/src/components/info-label/test/info-label.spec.tsx

```diff
@@ -0,0 +1,60 @@
+import { SpecPage } from '@stencil/core/internal';
+import { KvInfoLabel } from '../info-label';
+import { newSpecPage } from '@stencil/core/testing';
+import { KvTagLetter } from '../../tag-letter/tag-letter';
+
+describe('Info Label (unit tests)', () => {
+	let page: SpecPage;
+
+	describe('when uses default props', () => {
+		beforeEach(async () => {
+			page = await newSpecPage({
+				components: [KvInfoLabel],
+				html: `<kv-info-label label-text="Default" />`
+			});
+		});
+
+		it('should match the snapshot', () => {
+			expect(page.root).toMatchSnapshot();
+		});
+	});
+
+	describe('when renders with description and read more stuffs', () => {
+		beforeEach(async () => {
+			page = await newSpecPage({
+				components: [KvInfoLabel],
+				html: `<kv-info-label label-title="DESCRIPTION" description="Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book." description-height="34" description-collapsed-text="Read more" description-opened-text="Read less" />`
```

Comment:
since this is an interpolated / common string, you can break lines here for readability